lib/index.js

```diff
@@ -1,53 +1,65 @@
 let path = require("path");
-let { readFile } = require("./promisified-fs");
+let { readFile, stat } = require("./promisified-fs");
 let tree = require("./tree");
 
 module.exports = (pluginConfig, assetManager, { watcher }) => {
-	let copyAll = buildCopyAll(pluginConfig, assetManager);
+	buildCopyAll(pluginConfig, assetManager).then(copyAll => {
```

Comment:
FWIW, this now seems inconsistent: Early on we'd agreed to move async continuations to a separate line (i.e. `.\nthen`). Of course, given current realities, you might as well start using your own preferred style.